en/docs/develop/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -6860,12 +6860,16 @@ definitions:
         type: array
         items:
           type: string
-        example: ["Pizzashack", "Petstore"]
+        example: "Pizzashack", "Petstore"
```

Comment:
I guess we won't need this change?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1539/commits/4a50c9c70224e3984912567488eccd30a591a991